molecule/default/tests/test_default.py

```diff
@@ -25,11 +25,32 @@ def test_expected_files_are_present(host, filename):
     assert f.mode == 0o755
 
 
+@pytest.mark.parametrize(
+    "executable",
+    ["/usr/local/bin/terraform", "/usr/local/bin/terraform-docs"],
+)
+def test_tools_executable_architecture(host, executable):
+    """Verify that the installed tools have the appropriate architecture."""
+    command = f"file {executable}"
+    cmd = host.run(command)
+    assert cmd.rc == 0, f"Command {command} returned a non-zero exit code."
+    # host.system_info.arch will return either "x86_64" or "aarch64", whereas
+    # file prints out the architecture as "x86-64" or "aarch64".  The call to
+    # string.replace() is therefore necessary to match up these two
+    # conventions.
+    assert host.system_info.arch.replace("_", "-") in cmd.stdout
```

Comment:
Should we make this more specific to avoid any unintended replacements in the future?
```suggestion
    assert host.system_info.arch.replace("x86_64", "x86-64") in cmd.stdout
```